### ISSUE TITLE
[Bug]: Same arg in @subcommand function gives error

### ISSUE THREAD

**Issue by mischa2006** [KapiMC/Kapi#10]
### Checklist

- [x] I am using an up-to-date version.
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### Kapi Version

0.2.0

### Minecraft Version

1.20.1

### Description

when i have 2 commands with the same arguments, like for example from your example:
```
 @SubCommand
    public void add(CommandSender sender, @Literal("add") String add, int number) {
        int amount = 1;
        amount+= number;
        // Command logic
    }
    
    @SubCommand
    public void remove(CommandSender sender, @Literal("remove") String add, int number) {
        int amount = 1;
        amount-= number;
        // Command logic
    }
 ```

### Expected Behavior

It should let me create both subcommands, create a working command, and create the tab completion

### Steps to Reproduce

1. create command class with @subcommands
2. add 2 commands with the same arguments (in here the  `@Literal`  Annotation does nothing to stop this error)
3. launch the game lol

### Logs and Stack Traces

```
 [ViriAmes] An error occurred while loading the plugin!
[19:59:30] [Server thread/WARN]: [ViriAmes] Task #50 for ViriAmes v0.1.0 generated an exception
java.lang.IllegalArgumentException: Ambiguous methods remove and add in TestCommand class, methods with the same priority are not allowed!
        at io.github.kapimc.kapi.commands.CommandProcessor.process(CommandProcessor.java:78) ~[?:?]
        at io.github.kapimc.kapi.core.KapiPlugin.registerCommand(KapiPlugin.java:173) ~[?:?]
        at dev.nerds.viriames.ViriAmes.onPluginLoad(ViriAmes.java:85) ~[?:?]
        at io.github.kapimc.kapi.core.KapiPlugin.lambda$onEnable$0(KapiPlugin.java:58) ~[?:?]
        at io.github.kapimc.kapi.utility.TaskBuilder.lambda$create$0(TaskBuilder.java:83) ~[?:?]
        at io.github.kapimc.kapi.utility.TaskBuilder.lambda$schedule$1(TaskBuilder.java:111) ~[?:?]
        at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:415) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1256) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:387) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1197) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1014) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.20.1-R0.1-SNAPSHOT.jar:3871-Spigot-d2eba2c-3f9263b]
        at java.lang.Thread.run(Thread.java:1583) ~[?:?]
        ````

**Comment by Kyren223** [comment]
Sorry, I have been busy with other stuff, hopefully I will get a chance to fix it today and a fix (v0.2.1) should be out in less than a few days.

**Comment by mischa2006** [comment]
Great! take your time and lmk if i can help
